src/App.tsx

```diff
@@ -1,9 +1,23 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const changeQuery = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setQuery(event.target.value);
+  };
+
+  const visibleMovies = moviesFromServer.filter(({ title, description }) => {
+    const lowerTitle = title.toLowerCase();
+    const lowerDescription = description.toLowerCase();
+    const lowerQuery = query.toLowerCase();
+
+    return lowerTitle.includes(lowerQuery)
+      || lowerDescription.includes(lowerQuery);
```

Comment:
creating variables for improving readability is a great idea 🔥 

You can still make it better. Create booleans for `doesTitleMatch` and `doesDescriptionMatch` instead of those